modules/login/test/Request_Account_test_plan.md

```diff
@@ -8,7 +8,7 @@
 5. Verify "Return to Loris login page" link on Thank you page works
 	
 ### Approving new User Account Request:
-6. Log in as another user who has permission: user_accounts and does not have permission:user_accounts_multisite (User Management and Across all sites create and edit users). Verify that new account request notification is counted in Dashboard (count has incremented).
+6. Log in as another user who has permission: user_accounts (User Management) and does not have permission:user_accounts_multisite (Across all sites create and edit users). Verify that new account request notification is counted in Dashboard (count has incremented).
```

Comment:
6. Log in as another user who has permission: `user_accounts` (User Management) and does not have permission: `user_accounts_multisite` (Across all sites create and edit users). Verify that new account request notification is counted in Dashboard (count has incremented).

Reply:
Oops, sorry..